developer-docs-site/docs/sdks/ts-sdk/index.md

```diff
@@ -4,46 +4,117 @@ slug: "index"
 hidden: false
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ThemedImage from '@theme/ThemedImage';
+
 # Aptos TypeScript SDK
 
-Aptos provides a fully supported TypeScript SDK with the source code in the [Aptos-core GitHub](https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/typescript/sdk) repository. Much of the functionality of the TypeScript SDK can be found in the [Rust](../rust-sdk.md) and [Python](../python-sdk.md) SDKs. Nevertheless, Aptos strongly encourages you to use the TypeScript SDK for app development whenever possible.
+:::note
+This documentation is for TypeScript SDK > ` v2.0.0`. You can find the documentation for SDK `v1.x.x` [here](./v1/index.md)
+
+Looking to migrate to `TypeScript SDK v2`? check out the [migration guide](./migration-guide.md)
+:::
```

Comment:
Hmm... this should only be temporary while and we should ideally take it down in 2 weeks or so imo.

Reply:
you mean take this `note` down....?

Reply:
yeah, just a mental comment, so we do remove it...